MyClass.java

```diff
@@ -75,3 +114,4 @@ public static void row(int spaces, int stars){
 }
 }
 
+>>>>>>> bee68808de44320c3737a7c9146f8c7fa0194c59
```

Comment:
This line wont compile along with others like it. You'll need to make deletions of lines like this one for it to run properly.